assets/styles.css

```diff
@@ -0,0 +1,129 @@
+html{
+  width: 100vw;
+  height: 100vh;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: #E3EDF7;
+  font-family: 'Exo 2', sans-serif;
+}
+
+.keyboard {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 50%;
+}
+
+.keyboard__text-area {
+  width: 100%;
+  margin-bottom: 20px;
+}
+
+.keyboard__message {
+  margin-top: 20px;
+}
+
+.keyboard-panel {
+}
+
+.keyboard-line {
+  display: flex;
+  justify-content: space-between;
+}
+
+.keyboard-line__button {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: flex-end;
+  width: 50px;
+  height: 50px;
+  box-sizing: border-box;
+  padding: 5px;
+  user-select: none;
+  margin: 3px;
+  background: #E3EDF7;
+  box-shadow: 4px 4px 20px rgba(111, 140, 176, 0.5), -6px -6px 20px #FFFFFF, 2px 2px 4px rgba(114, 142, 171, 0.4);
+  border-radius: 10px;
+}
+.keyboard-line__button_single {
+  align-items: center;
+}
+
+.keyboard-line__button_single .keyboard-line__button-shifted-text{
+  display: none;
+}
+
+.keyboard-line__button:hover {
+  cursor: pointer;
+  background: #DBE6F0;
+}
+/* .keyboard-line__button:active, */

```

Comment:
No comments in the completed task.

Reply:
Remove commented style in commit below

https://github.com/BlackTruth/virtual-keyboard/commit/155ce0b54e3bcf888457e882c53fcf5462807995#diff-42cb364f9d4783c39009483c0ad61341